src/wmtk/TetMeshOperationExecutor.cpp

```diff
@@ -681,7 +633,20 @@ Tuple TetMesh::TetMeshOperationExecutor::split_edge()
     // return tuple
     // which one to return?
     // TODO
```

Comment:
Is this TODO and the subsequent (commented out) code still relevant? If not, please clean it up. If it is, I feel like there should be some sort of `spdlog::warn()` here to tell the user that the behavior of the code might change in the future. Otherwise, I am afraid that this TODO will be forgotten.